roles/solr/files/schema.xml

```diff
@@ -193,6 +193,13 @@
   <copyField source="dct_provenance_s"   dest="dct_provenance_sort"/>
   <copyField source="dc_publisher_s"     dest="dc_publisher_sort"/>
   <copyField source="dc_title_s"         dest="dc_title_sort"/>
+  
+    <!-- dl-beta sorting-->
+
+  <copyField source="dc_source_s"         dest="dc_source_sort"/>
+  <copyField source="dc_created_sm"       dest="dc_created_sort"/>
+  <copyField source="dc_date_uploaded_dt" dest="dc_date_uploaded_sort"/>
+  <copyField source="dc_date_modified_dt" dest="dc_date_modified_sort"/>
```

Comment:
Remove the blank lines and list the sort fields directly after the other ones.

Also, I don't think you need to copy the date fields into a sort field...they should sort just fine as dates.

And Does copying the `dc_created_sm`, a multivalued field, into `dc_created_sort` a single-valued field, work?